pki.spec

```diff
@@ -660,9 +669,9 @@ Provides:         bundled(resteasy-servlet-initializer)
 %if 0%{?rhel} && 0%{?rhel} >= 10
 Requires:         tomcat9 >= 1:9.0.62
 %else
-Requires:         tomcat >= 1:9.0.62
+Requires:         tomcat >= 1:10.1.24
 %endif
```

Comment:
This condition should be removed since PKI 11.8 will only work with Tomcat 10 on all platforms.